### DISCUSSION TITLE
How to increase the number of pages?

### DISCUSSION THREAD

**Discussion by Moyf** [CaiJimmy/hugo-theme-stack#1033] [Q&A] [answered]
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/4013062/4ebd4e23-3a18-4011-b4ed-e326cf63f6e4)
For now, it's `1 2 ... 5`

How to increase the number to like `1 2 3 4 5`

And only more then 5 pages, use `...` like: `1 2 3 ... 7 8`

**Comment by CaiJimmy** [comment, marked as answer]
Sorry for the late reply, but you need to modify this file: 

https://github.com/CaiJimmy/hugo-theme-stack/blob/2a18a92901e2689c23ff340c380a164a88208d78/layouts/partials/pagination.html#L11C17-L23C26

Replacing `else if` with `else` should do the job.

**Reply by Moyf** [reply]
Roger that, thank you !! ❤️️
